applications/compareRestart.hpp

```diff
@@ -0,0 +1,61 @@
+/*
```

Comment:
this means you create an UNSTReader.hpp with only its prototype, (i.e. this file but without the printHelp()) and an UNSTReader.cpp with the implementation. these should then be compiled into the opm-output library. finally you include UNSTReader.hpp in the test application.

usage of this in unit tests would correspond to my 'more than one application using the class'.